### DISCUSSION TITLE
Private Issues and PRs for public repos managed by private enterprise

### DISCUSSION THREAD

**Discussion by pragmaticivan** [community/community#5434] [Enterprise]
It would be great to have private issues and PRS that can only be seen by users belong to the org of the repo.

That's great for NDA data, SecOps reviews, etc...

**Comment by dregad** [comment]
Duplicate of #7971 ?

**Reply by pragmaticivan** [reply]
@dregad 7971 was created in `on Nov 21, 2021` this one was Sep 2, 2021. 7971 is the duplicated one.

**Reply by dregad** [reply]
Thanks for the nitpick :roll_eyes:
